doc/fq.1

```diff
@@ -0,0 +1,392 @@
+.TH FQ "1" "July 2023"
+
+.SH NAME
+fq \- jq for binary formats
+
+.SH SYNOPSIS
+\fBfq\fR [\fI\,OPTION\/\fR]... [--] \fI\,EXPR [\fI\,FILE\/\fR]...
```

Comment:
It would be nice to have some usage examples here i think. maybe same as shown for fq -h?